packages/realm-react/jest.config.json

```diff
@@ -5,7 +5,8 @@
     "<rootDir>/src"
   ],
   "setupFilesAfterEnv": [
-    "@testing-library/jest-native/extend-expect"
+    "@testing-library/react-native/extend-expect",
+    "<rootDir>/src/__tests__/setup.ts"
```

Comment:
I'm adding this new setup file to handle all global setup of `Realm` so we don't need to do this in the test files themselves.